docs-site/content/en/db/project-db-directory.md

```
@@ -0,0 +1,319 @@
+---
+title: Project DB Directory
+description: .moai/project/db/ template file set and customization guide
+weight: 40
+draft: false
+---
+
+## 7-File Template Set
+
+When you run `/moai db init`, these 7 files are automatically created in `.moai/project/db/`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)